packages/storage/.eslintrc.js

```diff
@@ -14,6 +33,13 @@ module.exports = {
         varsIgnorePattern: '^_',
         args: 'none'
       }
+    ],
+    'import/no-extraneous-dependencies': [
```

Comment:
We're not, accidental copy-paste artifact. I only put this here because vscode for some reason doesn't process the top-level `packageDir` path correctly, only if it's in the package's local `eslintrc.js` file.

Reply:
Do you mean we need it here to make the eslint plugin happy?

Reply:
Yes.